packages/apps/proof-of-us/src/utils/getIPFSLink.ts

```diff
@@ -2,10 +2,10 @@ import { env } from './env';
 
```

Comment:
removing all the extra data in `proofs`.
this was needed for having a leaderboard